mindsdb/utilities/wizards.py

```diff
@@ -227,8 +227,13 @@ def daemon_creator(python_path, config_path=None):
     return daemon_path
 
 
-def make_executable(python_path, exec_path, config_path=None):
+def make_executable(python_path, exec_path, config_path=None, update=False):
     text = f"""#!/bin/bash
+update={update}
+if [ "$update" = True ] 
```

Comment:
`True` should be `true`, and the value of `update` should be lowercased so it's a valid shell boolean value. Other than that looks good.

Reply:
done!